### PR TITLE
Fabo/1573 sent errors via raven

### LINKED ISSUE #1573: We should make sure, that the resulting errors appear in some log and are sent via Raven. So that we can act on odd behavior with remote clients.
> We should make sure, that the resulting errors appear in some log and are sent via Raven. So that we can act on odd behavior with remote clients.

_Originally posted by @faboweb in https://github.com/cosmos/voyager/pull/1569_

### PR DESCRIPTION
**Pull request by faboweb**
Closes #1573 

_Description:_

❤️ Thank you!

---

* [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
* [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: app/src/renderer/vuex/modules/connection.js [hunk 1]
```diff
@@ -92,6 +91,7 @@ export default function({ node }) {
         { query: `tm.event = 'NewBlockHeader'` },
         (err, event) => {
           if (err) {
+            Raven.captureException(err)
```
Comment (fedekunze) [inline app/src/renderer/vuex/modules/connection.js]: `error`

File: app/src/renderer/vuex/modules/connection.js [hunk 2]
```diff
@@ -148,7 +148,11 @@ export default function({ node }) {
         }
       }, timeout)
       node.rpc.status(err => {
-        if (err) return console.error(`Couldn't get status via RPC:`, err)
+        if (err) {
```
Comment (fedekunze) [inline app/src/renderer/vuex/modules/connection.js]: ditto

File: app/src/renderer/vuex/modules/delegation.js [hunk 3]
```diff
@@ -145,6 +145,7 @@ export default ({ node }) => {
           title: `Error fetching delegations`,
           body: err.message
         })
+        Raven.captureException(err)
```
Comment (fedekunze) [inline app/src/renderer/vuex/modules/delegation.js]: ditto

**Comment by faboweb** [comment]
done @fedekunze

**Comment by faboweb** [comment]
boy, I don't want to refactor the whole repo in this PR ;P

**Pull request merged**